distribution/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -138,3 +138,820 @@ For example, if a Spatial search and Contextual search are the components of a C
+[source,json,linenums]
+----
+"data": {
+	"count": 250,
+	"format": "geojson",
+	"id": "4303ba5d-21af-4878-9a4c-808e80052e6c",
+	"cql": "INTERSECTS(anyGeo, POLYGON ((-112.7786 32.2159, -112.7786 45.6441, -83.7297 45.6441, -83.7297 32.2159, -112.7786 32.2159)))",
+	"start": 1
+}
+----
+
+==== Query Responses
+
+===== Query Response Channel
+
+The query responses are returned on the `/<id>` channel, which should be subscribed to in order to retrieve the results.
+Replace `<id>` with the id that was used in the request.
+The <<Asynchronous Search and Retrieval>> section details how to subscribe to a CometD channel.
+
+===== Query Response Message Format
+
+The response is returned as a data map that contains an internal map with the following keys:
+
+[cols="1m,3,1,1,1" options="header"]
+|===
+|Map Key
+|Description
+|Value Type
+|Possible Values
+|Example Values
+
+|id
+|ID that corresponds to the request
+|String
+|ID value
+|
+
+|hits
+|Total number of query hits that were found by the server
+|Number
+|Integer >= 0
+|This contains the total amount of items that were found by the query. Depending on the 'count' in the request, not all of the results may be returned.
+
+|results
+|Array of metacard results
+|Array of Maps
+|GeoJson-formatted value
+|This format is defined by the GeoJSON Metacard Transformer.
+
+|results/metacard/actions
+|An array of actions that applies to each metacard, injected into each metacard
+|Array of Maps
+|Array of objects, possibly empty if no actions are available
+|Each Action will contain an id, title, description, and url
+
+|status
+|Array of status for each source queried
+|Array
+|
+|
+
+|status.state
+|Specifies the state of the query
+|String
+|SUCCEEDED, FAILED, ACTIVE
+
+|status.elapsed
+|Time in milliseconds that it took for the source to complete the request
+|Number
+|Integer >= 0
+|
+
+|status.hits
+|Number of records that were found on the source that matched the query
+|Number
+|Integer >= 0
+|
+
+|status.id
+|ID of the federated source
+|String
+|Any string value
+|
+
+|status.results
+|Number of results that were returned in this response from the source
+|Number
+|Integer >= 0
+|
+
+|types
+|A Map mapping a metacard-type's name to a map about that metacard-type. Only metacard-types represented by the metacards returned in the query are represented.
+|Map of Maps
+|
+|The Map defining a particular `metacard-type` maps the fields supported by that `metacardtype` to the datatype for that particular field.
+|===
+
+====== Query Response Examples
+
+.Example Query Response
+[source,json,xml]
+----
+"data": {
+	"id": "4303ba5d-21af-4878-9a4c-808e80052e6c",
+	"hits": 20,
+ 	"results": [
+ 	{
+		"metacard": {
+ 			"geometry": {
+ 				"coordinates": [ 174.77557, -41.28664 ],
+		 		"type": "Point"
+ 			},
+			"properties": {
+				"created": "2014-07-02T18:53:59.496 +0000",
+ 				"id": "126e16d340dd47b7ad823b70662ed8ca",
+				"metacard-type": "${ddf-branding-lowercase}.metacard",
```

Comment:
I always locally build and review the documents that are generated when I review a documentation PR... I can confirm that these resolve correctly... question is, I'm not sure that @oconnormi wasn't aware of this, so is he actually trying to argue that, even though variables can be used in these cases, hard coding "ddf" is more correct in these situations?